mlflow/pyfunc/__init__.py

```diff
@@ -460,7 +460,7 @@ def predict(*args):
 
 
 def save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,
-               model=Model(), python_model=None, artifacts=None):
+               model=Model(), python_model=None, artifacts=None, **kwargs):
```

Comment:
I like the `**kwargs` approach. For API consistency with other `save_model` methods, can we rename the `model` keyword argument to `mlflow_model` (like you had it before) and check for `model` (the deprecated argument name) in `kwargs` instead?

In this case, can we also update the documentation to read `:param mlflow_model: ...` instead of `:param model: ...`?